specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/EdgeSubscription.json

```diff
@@ -0,0 +1,398 @@
+    "paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.AzureStack/edgeSubscriptions": {
+        "get": {
+          "x-ms-examples": {
+            "Returns a list of all edge suscriptions.": {
+              "$ref": "examples/EdgeSubscription/ListEdgeSubs.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Returns a list of all edge subscriptions.",
+          "operationId": "EdgeSubscriptions_List",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+              "$ref": "#/parameters/DeviceIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscriptionList"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
+        }
+      },
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.AzureStack/edgeSubscriptions/{edgeSubscriptionName}": {
+        "get": {
+          "x-ms-examples": {
+            "Returns the properties of an edge suscription.": {
+              "$ref": "examples/EdgeSubscription/GetEdgeSub.json"
+            }
+          },
+          "tags": [
+            "EdgeSubscriptions"
+          ],
+          "description": "Returns the properties of an Azure Stack edge subscriptions.",
+          "operationId": "EdgeSubscriptions_Get",
+          "parameters": [
+            {
+              "$ref": "#/parameters/SubscriptionIdParameter"
+            },
+            {
+              "$ref": "#/parameters/ResourceGroupNameParameter"
+            },
+            {
+                "$ref": "#/parameters/EdgeSubscriptionNameParameter"
+            },
+            {
+              "$ref": "#/parameters/ApiVersionParameter"
+            }
+          ],
+          "responses": {
+            "200": {
+              "description": "OK",
+              "schema": {
+                "$ref": "#/definitions/EdgeSubscription"
+              }
+            },
+            "default": {
+              "description": "Error response describing why the operation failed.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
+            }
+          },
+          "x-ms-pageable": {
+            "nextLinkName": "nextLink"
+          }
```

Comment:
There shouldn't be paging on single GET. (Copy paste from list call?)

Reply:
Likely got copied, removed it now.

Reply:
I have responded to your comments, can you please take a look? @majastrz